app/templates/components/banner.html

```diff
@@ -16,14 +16,14 @@ <h1 class="banner-title">{{ subhead }}</h1>
     </p>
     {% endif %}
     {% if delete_button %}
-      <form method='post'>
+      <form method='post' {% if action %}action="{{action}}"{% endif %}>
```

Comment:
Would be better to use:
```suggestion
      {% call form_wrapper(action=action) %}
```